docs/tutorials/mobile/part-3.md

```diff
@@ -0,0 +1,826 @@
+---
+title: 'Part 3: Create the game pages'
+authors: 'Benjamin Fuentes'
+last_update:
+  date: 12 December 2023
+---
+
+In this section, you will create the pages to :
+
+- Create a game.
+- Join a game.
+- Play a session.
```

Comment:
I don't feel like i understand what the pages do; maybe we could have a description of the flow of the application, like what happens in a numbered list of steps? I don't know what a "session" is here.

Reply:
Well, I will add more details in advance. The flow demo/test is done after the copy/paste of the code